src/ostorlab/agent/agent.py

```diff
@@ -242,8 +242,8 @@ def process_message(self, selector: str, message: bytes) -> None:
             system_info = system.get_system_info()
             if system_info is not None:
                 logger.error("System Info: %s", system_info)
-            logger.error("Message: %s", object_message)
             logger.exception("Exception: %s", e)
+            logger.error("Message: %s", object_message.selector)
```

Comment:
As discussed, we need a method to recursively process the data to show a portion if bigger than max size.